app/src/main/java/pm/gnosis/heimdall/ui/base/ViewModelActivity.kt

```diff
@@ -20,4 +24,12 @@ abstract class ViewModelActivity<VM : ViewModel> : BaseActivity() {
         inject(viewComponent())
         setContentView(layout())
     }
+
+    protected fun colorStatusBar() {
```

Comment:
Maybe we could make the color a parameter of the function, but not really required right now

Reply:
done